From 95511a2a92ab95f75734a7460325887daa93cbcd Mon Sep 17 00:00:00 2001
From: cjihrig <cjihrig@gmail.com>
Date: Tue, 23 Jan 2018 11:57:08 -0500
Subject: [PATCH] SSO-202 Upgrade sdcsso to Node 8 (sdcsso zone setup requires
 postfix)

Reviewed by: Trent Mick <trent.mick@joyent.com>
Approved by: Trent Mick <trent.mick@joyent.com>
---
 targets.json.in | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/targets.json.in b/targets.json.in
index 2ac3cb4..44d2944 100644
--- a/targets.json.in
+++ b/targets.json.in
@@ -1374,7 +1374,8 @@ cat <<EOF
     "// image_uuid": "triton-origin-multiarch-15.4.1@1.0.1",
     "image_uuid": "04a48d7d-6bb5-4e83-8c3b-e60a99e0f48f",
     "pkgsrc": [
-      "openssl-1.0.2k"
+      "openssl-1.0.2k",
+      "postfix-3.0.2nb2"
     ],
     "repos": [
         {"url":  "git@github.com:joyent/sdcsso.git" }
-- 
2.21.0

